addons/rook/template/testgrid/k8s-docker.yaml

```diff
@@ -199,60 +103,12 @@
       s3Override: "__testdist__"
       isSharedFilesystemDisabled: true
       isBlockStorageEnabled: true
+  unsupportedOSIDs:
+    - centos-74 # Rook > 1.7 not supported on 3.10 kernel
```

Comment:
is there docs somewhere to back this up?

Reply:
I had glanced at this [issue](https://github.com/rook/rook/issues/9514#issuecomment-1006448126) and presumed the kernel wasn't supported. However, after giving this more thought, this may not necessarily be true. It could have meant that the 3.10 kernel needed the RBD kernel module loaded. I'll test again.

Ceph doesn't actually publish kernel versions it does _not_ support but recommends a list kernel [versions](https://docs.ceph.com/en/latest/start/os-recommendations/#linux-kernel) that should support Ceph features.

Reply:
I know that the rhel 3.10 kernel is heavily patched and typically works with most new software.

Reply:
On `Centos-7.4.1708` (version we use for testgrid) which has a kernel version of `3.10.0-693.el7.x86_64`, the Rook 1.8 `csi-rbdplugin` container fails to become Ready because of the following error:
```
W0930 19:25:01.937253    9535 rbd_attach.go:209] rbd-nbd: nbd modprobe failed with error an error (exit status 1) occurred while running modprobe args: [nbd]
E0930 19:25:01.955710    9535 rbd_util.go:248] reading file "/sys/bus/rbd/supported_features" failed: open /sys/bus/rbd/supported_features: no such file or directory
F0930 19:25:01.955726    9535 driver.go:148] open /sys/bus/rbd/supported_features: no such file or directory
goroutine 1 [running]:
k8s.io/klog/v2.stacks(0x1)
	/go/src/github.com/ceph/ceph-csi/vendor/k8s.io/klog/v2/klog.go:1038 +0x8a
k8s.io/klog/v2.(*loggingT).output(0x322f0c0, 0x3, 0x0, 0xc0005a6380, 0x1, {0x25ad879, 0x10}, 0xc000100000, 0x0)
	/go/src/github.com/ceph/ceph-csi/vendor/k8s.io/klog/v2/klog.go:987 +0x5fd
k8s.io/klog/v2.(*loggingT).printDepth(0xc0005ba080, 0x3f, 0x0, {0x0, 0x0}, 0x3, {0xc000436690, 0x1, 0x1})
	/go/src/github.com/ceph/ceph-csi/vendor/k8s.io/klog/v2/klog.go:735 +0x1ae
k8s.io/klog/v2.FatalDepth(...)
	/go/src/github.com/ceph/ceph-csi/vendor/k8s.io/klog/v2/klog.go:1518
github.com/ceph/ceph-csi/internal/util/log.FatalLogMsg({0xc0005ba040, 0x7fff88a4807a}, {0x0, 0x0, 0x7fff88a4809d})
	/go/src/github.com/ceph/ceph-csi/internal/util/log/log.go:59 +0x9e
github.com/ceph/ceph-csi/internal/rbd/driver.(*Driver).Run(0xc0005f75c0, 0x322f2e0)
	/go/src/github.com/ceph/ceph-csi/internal/rbd/driver/driver.go:148 +0xa25
main.main()
	/go/src/github.com/ceph/ceph-csi/cmd/cephcsi.go:226 +0x991

goroutine 19 [chan receive]:
k8s.io/klog/v2.(*loggingT).flushDaemon(0x0)
	/go/src/github.com/ceph/ceph-csi/vendor/k8s.io/klog/v2/klog.go:1181 +0x6a
created by k8s.io/klog/v2.init.0
	/go/src/github.com/ceph/ceph-csi/vendor/k8s.io/klog/v2/klog.go:420 +0xfb

goroutine 38 [sleep]:
time.Sleep(0x6fc23ac00)
	/usr/local/go/src/runtime/time.go:193 +0x12e
sigs.k8s.io/controller-runtime/pkg/log.init.0.func1()
	/go/src/github.com/ceph/ceph-csi/vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:63 +0x38
created by sigs.k8s.io/controller-runtime/pkg/log.init.0
	/go/src/github.com/ceph/ceph-csi/vendor/sigs.k8s.io/controller-runtime/pkg/log/log.go:62 +0x25

goroutine 65 [IO wait]:
internal/poll.runtime_pollWait(0x7f1162ef2090, 0x72)
	/usr/local/go/src/runtime/netpoll.go:234 +0x89
internal/poll.(*pollDesc).wait(0xc000218300, 0x42120f, 0x0)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:84 +0x32
internal/poll.(*pollDesc).waitRead(...)
	/usr/local/go/src/internal/poll/fd_poll_runtime.go:89
internal/poll.(*FD).Accept(0xc000218300)
	/usr/local/go/src/internal/poll/fd_unix.go:402 +0x22c
net.(*netFD).accept(0xc000218300)
	/usr/local/go/src/net/fd_unix.go:173 +0x35
net.(*UnixListener).accept(0xc0005b2088)
	/usr/local/go/src/net/unixsock_posix.go:167 +0x1c
net.(*UnixListener).Accept(0xc0005f7830)
	/usr/local/go/src/net/unixsock.go:260 +0x3d
google.golang.org/grpc.(*Server).Serve(0xc000384700, {0x20b7cf0, 0xc0005f7830})
	/go/src/github.com/ceph/ceph-csi/vendor/google.golang.org/grpc/server.go:779 +0x362
github.com/ceph/ceph-csi/internal/csi-addons/server.(*CSIAddonsServer).serve(0xc00015fd80, {0x20b7cf0, 0xc0005f7830})
	/go/src/github.com/ceph/ceph-csi/internal/csi-addons/server/server.go:117 +0x7f
created by github.com/ceph/ceph-csi/internal/csi-addons/server.(*CSIAddonsServer).Start
	/go/src/github.com/ceph/ceph-csi/internal/csi-addons/server/server.go:106 +0x2ef
```

Which leads me to believe that Rook CSI does not support this particular kernel. I have excluded CentOS-7.4 from the testgrid matrix because of this.